src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: " + calculateArea() + " sq. units, "
+                + "radius: " + radius + " units, "
+                + "color: " + getColor();
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw(Figure figure) {
+        System.out.println(figure);
+    }
+}

```

Comment:
The class 'Circle' should not extend 'Figure' just to share behavior. Consider using an interface to define common behavior and separate the implementation into their respective classes.